### DISCUSSION TITLE
Works great with B1A24S15P

### DISCUSSION THREAD

**Discussion by killee** [syssi/esphome-jk-bms#4] [Show and tell]
Thank you for your work!

It was easy to get it working with B1A24S15P.

H8.X__S8.1.0H__
BT3072020120000200521001

Regards
Kilian

**Comment by syssi** [comment]
Thanks for your feedback! I will add the model to the list of supported devices.

**Reply by syssi** [reply]
Could you do me a favour? Next to the TTL connector should be a version number printed to the PCB. Which hardware version do you own?

**Reply by killee** [reply]
I'm Sorry, but I didn't open the case yet. I'll do this in the next days.

**Reply by syssi** [reply]
Ah, got it. Your version is the one with the black enclosure.

**Reply by killee** [reply]
Thats right: https://de.aliexpress.com/item/4000529723243.html

**Comment by rem72** [comment]
Hi Syssi, I own a JK-B2A24S20P and guess if this can work with it.
But I have one doubt. Is a ttl level converter to 3.3 volt needed between the jk port and the esp32?

**Reply by syssi** [reply]
IMO there is no converter needed because both devices has the same 3.3V level. You could try to measure the voltage to be sure.
